src/Cli.Tests/EndToEndTests.cs

```diff
@@ -746,6 +744,40 @@ string options
         process.Kill();
     }
 
+    [DataRow("", "--version", false, DisplayName = "Checking dab version with --version.")]
+    [DataRow("", "--help", false, DisplayName = "Checking version through --help option.")]
+    [DataRow("edit", "--new-option", false, DisplayName = "Version printed with invalid command edit.")]
+    [DataTestMethod]
+    public void InvalidCliVerbsAndOptions_DisplayVersionWithCommitHashAndConfigFileName(
+    string command,
+    string options,
+    bool isParsableDabCommandName)
```

Comment:
since all the cases have `isParsableDabCommandName` as `false` do we really need it as a variable in the data rows?

Reply:
nope, good catch, fixed.